data/pipelines.py

```diff
@@ -0,0 +1,118 @@
+from collections import Counter
+
+from nltk.corpus import stopwords
+from nltk.stem import PorterStemmer
+from nltk.stem import WordNetLemmatizer
+from nltk.tokenize import word_tokenize
+
+from data.import_data import import_data
+
+
+def tokenize_pipe(prior):
+    for s in prior:
+        yield word_tokenize(s)
+
+
+def lower_pipe(prior):
+    for s in prior:
+        yield s.lower()
+
+
+def strip_stopwords_pipe(prior):
+    extra_stopwords = [
+        ',',
+        '.',
+        ';',
+        '``',
+        '\'',
+        '\'\'',
+        'would',
+        'could',
+        '\'s',
+        'one']
+
+    stops = [*stopwords.words('english'), *extra_stopwords]
+    for s in prior:
+        yield [
```

Comment:
Is it worth adding string.punctuation into this aswell as the stopwords list does not include punctuation? Could add this line in s = s.translate(None, string.punctuation) in here.

Reply:
No, some punctuation is pretty unique.